webpack/stories/index.js

```diff
@@ -11,6 +11,7 @@ import addingDependencies from './docs/addingDependencies.md';
 import internationalization from './docs/internationalization.md';
 import plugins from './docs/plugins.md';
 import SlotAndFill from './docs/SlotAndFill.md';
+import LegacyJs from './docs/LegacyJS.md';
```

Comment:
should be `./docs/LegacyJs.md`

Reply:
Thanks, I renamed it and forgot to update the import...